sqlmesh/utils/date.py

```diff
@@ -256,14 +256,20 @@ def to_ds(obj: TimeLike) -> str:
     return to_ts(obj)[0:10]
 
 
-def to_ts(obj: TimeLike) -> str:
+def to_ts(obj: TimeLike, include_fractional_seconds: bool = True) -> str:
```

Comment:
This feels like a mouthful. Can we just call this `include_microseconds`?

Reply:
what if it's precision?

Reply:
fine with include_microseconds as well